pkg/pgmodel/ingestor/metric_batcher.go

```diff
@@ -72,7 +112,9 @@ func runMetricBatcher(conn pgxconn.PgxConn,
 		var err error
 		tableName, err = initializeMetricBatcher(conn, metricName, completeMetricCreationSignal, metricTableNames)
 		if err != nil {
-			firstReq.reportResult(fmt.Errorf("initializing the insert routine has failed with %w", err))
+			err := fmt.Errorf("initializing the insert routine for metric %v has failed with %w", metricName, err)
+			log.Error("msg", err)
```

Comment:
Won't this log the same error twice, which was already being logged in the line below?

Reply:
No, the next line will return an error for the request but not necessarily log it.